app/views/admin/project_logs/_log_list.html.slim

```diff
@@ -30,31 +33,35 @@
             .item
               i.fa.fa-commenting-o>
               = log.progress_metric || t('project_step.no_status')
-            - unless params[:org].present? || params[:loan].present? || params[:step].present?
+
+            - unless @context.present? && @context == "calendar"
+              - unless params[:org].present? || params[:loan].present? || params[:step].present?
+                .item
+                  a href=admin_organization_path(step.project.organization)
+                    i.fa.fa-building-o>
+                    = step.project.organization.name
+              - unless params[:step].present?
+                .item
+                  a href=(admin_loan_path(step.project) + "/timeline-table#step-#{step.id}")
+                    = project_step_icon(step)
+                    = step.summary.present? ? step.summary : t('log.untitled_step')
```

Comment:
Here you are checking if summary is present but not above. Is that on purpose?

I like to put this kind of logic into the model with a method like `summary_or_untitled`. Then you can reuse it wherever you want a friendly display of the summary. Else you end up duplicating this same logic all over.